crates/air-lib/test-utils/src/test_runner.rs

```diff
@@ -97,17 +112,19 @@ pub fn create_avm(
     call_service: CallServiceClosure,
     current_peer_id: impl Into<String>,
 ) -> TestRunner {
-    let air_wasm_path = PathBuf::from("../target/wasm32-wasi/debug/air_interpreter_server.wasm");
-    let current_peer_id = current_peer_id.into();
-    let logging_mask = i32::MAX;
+    static POOL_CELL: OnceCell<object_pool::Pool<AVMRunner>> = OnceCell::new();
+
+    let pool = POOL_CELL.get_or_init(|| {
+        object_pool::Pool::new(
+            // We create an empty pool and let it fill on demand.
```

Comment:
According to coding standard used here one-sentence comment starts with non-capital letter and a dot isn't set at the end. Doc comments start with capital letter and ending with a dot.